### PR TITLE
TF-IDF and label noise

### PR DESCRIPTION
**Pull request by benleetownsend**

### CODE REVIEW

File: enso/experiment/__init__.py [hunk 1]
```diff
@@ -239,11 +253,25 @@ class Experiment(BaseObject):
...
         Instantiate a new experiment
         """
         super().__init__(*args, **kwargs)
+        self.resampler_ = resampler
+        self.auto_resample_ = auto_resample
+
+    def resample(self, X, y):
+        return self.resampler_.resample(X, y)
+
+    def __getattr__(self, item):
```
Comment (madisonmay) [inline enso/experiment/__init__.py]: Yeah let's not do this.

Comment (madisonmay) [reply enso/experiment/__init__.py]: There has to be a more transparent way.

File: enso/experiment/finetuning.py [hunk 2]
```diff
@@ -37,6 +53,127 @@ def predict(self, X, **kwargs):
...
+
+@Registry.register_experiment(ModeKeys.CLASSIFY, requirements=[("Featurizer", "PlainTextFeaturizer")])
+class Finetune10(Finetune):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model.config.trainable_layers = [True] * 10
+        self.model.config.trainable_old_embeddings = False
+        self.model.config.trainable_new_embeddings = False
+        self.model.config.init_embeddings_from_file = "embeddings.npy"
+
```
Comment (lhz1029) [inline enso/experiment/finetuning.py]: Do we actually want all the of these experiments in the registry, or were they mainly for a one-time thing?

File: enso/experiment/finetuning.py [hunk 3]
```diff
@@ -50,6 +187,235 @@ def predict(self, X, **kwargs):
...
+                fp.write(str(res.trainable_layers) + "\n")
+            self.model = Classifier(res)
+            self.model.fit(X, Y=y)
+
+        def predict(self, X, **kwargs):
+            """Predict results on test set based on current internal model."""
+            preds = self.model.predict_proba(X)
+            return pd.DataFrame.from_records(preds)
+
+
```
Comment (lhz1029) [inline enso/experiment/finetuning.py]: Same here. Do we want to merge these experiments into master?

**Comment by madisonmay** [comment]
Closing this as I don't think it's relevant anymore
